spec/concurrent/actor_spec.rb

```diff
@@ -148,7 +148,7 @@ def on_message(message)
           end
         end
 
-        it 'terminates with all its children', notravis: true do
```

Comment:
I wonder if this tag was somehow avoiding running transient tests in CI.
Maybe the failure in https://github.com/ruby-concurrency/concurrent-ruby/runs/5553488676?check_suite_focus=true is related (but maybe not)

Reply:
Given the Rakefile above it sounds like they were excluded, even on GitHub Actions before this PR